client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.css

```diff
@@ -0,0 +1,19 @@
+.opened-rma {
```

Comment:
You can refer the some utilities that are available here:

https://getbootstrap.com/docs/4.0/utilities/spacing/